.gitignore

```diff
@@ -1,7 +1,6 @@
 .idea
 bower_components
 node_modules
-client*
```

Comment:
client folder still need to be ignored in git push right only client.zip  to be avoided in gitignore?

Reply:
yes we can do that